nailgun/entities.py

```diff
@@ -4005,9 +4006,12 @@ def update(self, fields=None):
 
     def update_payload(self, fields=None):
         """Wrap submitted data within an extra dict."""
-        return {
-            u'organization': super(Organization, self).update_payload(fields)
-        }
+        org_payload = super(Organization, self).update_payload(fields)
+        payload = {u'organization': org_payload}
+        if 'redhat_repository_url' in org_payload:
+            rh_repo_url = org_payload['redhat_repository_url']
```

Comment:
What do you mean, who can use it? You aren't touching entity here, you're just updating the payload to be send. And response isn't affected as well - result is being processed in `update_json()` and then entity generated in `update()`; `update_payload()` is only responsible for what to send in request. Or i didn't get you :)